compute/cker/include/cker/eigen/depthwise_conv_op.h

```diff
@@ -0,0 +1,956 @@
+              const auto p = Eigen::internal::ploadu<Packet>(in + d);
+              Eigen::internal::pstoreu<T>(in_buf, p);
+              in_buf += kPacketSize;
+            }
+            for (int64_t d = 0; d < input_scalar_size; ++d)
+            {
+              T v = in[input_vectorized_size + d];
+              in_buf[d] = v;
+            }
+            in_buf += input_scalar_size;
+
+            // Pad the remainder of the output to vector register boundary.
+            for (int64_t d = 0; d < output_pad_size; ++d)
+            {
+              in_buf[d] = static_cast<T>(0);
+            }
+            in_buf += output_pad_size;
+          }
+          else
+          {
+            // Zero pad.
+            memset(in_buf, 0, sizeof(T) * padded_filter_inner_dim_size);
+            in_buf += padded_filter_inner_dim_size;
+          }
+        }
+      }
+    }
+  }
+};
+
+} // namespace functor
+} // namespace depthwise_conv_op
+} // namespace cker
+} // namespace nnfw
+
+// From tensorflow/core/kernels/depthwise_conv_grad_op.cc
+namespace nnfw
+{
+namespace cker
+{
+namespace depthwise_conv_op
+{
+
+// Enable CPUDevice only for depthwise_conv_op
+using CPUDevice = Eigen::ThreadPoolDevice;
+
+// Copies data from local region in 'out_backprop' into 'buffer'.
+// The local region coordinates are calculated as the set of output points which
+// used the input point ('in_r', 'in_'c') as input during the forward pass.
+// Rather than spatially reversing the filter, the input is reversed during
+// the copy. The copied data is padded to vector register-width boundaries so
+// that it is aligned for efficient traversal and vector multiply-add by the
+// depthwise input kernel.
+//
+// EX:
+//   in_depth = 3, depth_multiplier = 2, filter [2, 2], register_width = 4
+//
+//   'out_backprop': [batch, out_rows, out_cols, out_depth]
+//
+//     [a00, a01, a10, a11] [a20, a21, b00, b01]
+//     [b10, b11, b20, b21] [...]
+//     [e00, e01, e10, e11] [e20, e21, f00, f01]
+//     [f10, f11, f20, f21] [...]
+//
+//   'buffer' (register boundaries shown):
+//
+//     [f00, f01, f10, f11] [f20, f21, 0, 0]   in_row = 0, in_col = 0
+//     [e00, e01, e10, e11] [e20, e21, 0, 0]   in_row = 0, in_col = 1
+//     [b00, b01, b10, b11] [b20, b21, 0, 0]   in_row = 1, in_col = 0
+//     [a00, a01, a10, a11] [a20, a21, 0, 0]   in_row = 1, in_col = 1
+//
+template <typename T>
+static void CopyOutputBackpropRegion(int, int, int, int, int filter_rows_, int filter_cols_, int,
+                                     int stride_, int pad_rows_, int pad_cols_, int out_rows_,
+                                     int out_cols_, int out_depth,
+                                     const int64_t padded_filter_inner_dim_size, const int64_t in_r,
+                                     const int64_t in_c, const T *out_backprop, T *buffer)
+{
+  typedef typename Eigen::internal::packet_traits<T>::type Packet;
+  static const int64_t kPacketSize = (sizeof(Packet) / sizeof(T));
```

Comment:
How about using local variable instead of using static variable?
```suggestion
  const int64_t kPacketSize = (sizeof(Packet) / sizeof(T));
```

Reply:
Also, you can replace this with an inline namespace variable since C++17.

Reply:
I checked that this code was from tensorflow. You can ignore the above comment.

Reply:
Yes, this code came from TensorFlow. I want to keep the existing code. :)